mantis-control-plane/mantis-control-plane-server/build.gradle

```diff
@@ -35,6 +35,10 @@ ext {
 dependencies {
     api project(':mantis-control-plane:mantis-control-plane-core')
 
+    implementation platform('software.amazon.awssdk:bom:2.23.15')
+    implementation 'software.amazon.awssdk:dynamodb'
```

Comment:
I feel like it's better to move this to its own module so we can avoid adding dynamodb SDK dep to the control-plane-server everywhere?

Reply:
@Andyz26 I agree, let me get this moved into a new module today.